benchmarks_gui/CMakeLists.txt

```diff
@@ -41,13 +51,17 @@ include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 link_directories(${Boost_LIBRARY_DIRS})
 link_directories(${catkin_LIBRARY_DIRS})
 
-qt4_wrap_cpp(MOC_FILES include/main_window.h include/trajectory.h include/benchmark_processing_thread.h)
-qt4_wrap_ui(UIC_FILES
-  src/ui/main_window.ui
+set(MOC_FILES include/main_window.h include/trajectory.h include/benchmark_processing_thread.h)
```

Comment:
Having `CMAKE_AUTOMOC == ON` should suffice. I guess MOC_FILES doesn't need to be used anymore.